webtau-standalone-runner-groovy/src/main/groovy/com/twosigma/webtau/runner/standalone/StandaloneTestRunner.groovy

```diff
@@ -46,12 +48,14 @@ class StandaloneTestRunner {
         this.isTerminated = new AtomicBoolean(false)
     }
 
-    void process(Path scriptPath, delegate) {
+    void process(TestFile testFile, delegate) {
+        Path scriptPath = testFile.path
         currentTestPath = scriptPath.isAbsolute() ? scriptPath : workingDir.resolve(scriptPath)
+        currentShortContainerId = testFile.shortContainerId
```

Comment:
containerId makes sense in html report context. Not sure if it is informative enough in TestFile context

Reply:
Yeah, so I initially had it as shortFileName in TestFile and then it was changed into shortContainerId when throwing stuff over to JS.  I then decided to make it consistent across the board.  Happy to go back, not sure I feel strongly about it either way.